docker/test_env/besu.go

```diff
@@ -0,0 +1,211 @@
+package test_env
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	tc "github.com/testcontainers/testcontainers-go"
+	tcwait "github.com/testcontainers/testcontainers-go/wait"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/docker"
+	"github.com/smartcontractkit/chainlink-testing-framework/logging"
+	"github.com/smartcontractkit/chainlink-testing-framework/utils/testcontext"
+)
+
+const (
+	//TODO use Tate's mirror?
+	BESU_IMAGE_TAG = "23.10"
+)
+
+type Besu struct {
+	EnvComponent
+	ExternalHttpUrl      string
+	InternalHttpUrl      string
+	ExternalWsUrl        string
+	InternalWsUrl        string
+	InternalExecutionURL string
+	ExternalExecutionURL string
+	generatedDataHostDir string
+	chainConfg           *EthereumChainConfig
+	consensusLayer       ConsensusLayer
+	l                    zerolog.Logger
+	t                    *testing.T
```

Comment:
is it necessary to have *testing.T ? If someone wants to use it in some scripts other than tests, depending on *testing.T might not be ideal.

Reply:
unfortunately it is, re: thread in Slack